app/views/items/edit.html.erb

```diff
@@ -23,6 +23,16 @@
       <%= f.text_field :description, class: "form-control" %>
     </div>
 
+    <div class="form-group">
+      <%= f.label 'SKU:' %>
+      <%= f.text_field :sku, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= f.label 'Price:' %>
+      <%= f.text_field :price, value: number_to_currency(@item.price, precision: 2), class: "form-control", data: { guard: "moneyUS" } %>
```

Comment:
Would this perhaps be better as [float guard](http://guardsjs.com/named_guards.html#float) and using [input groups](http://getbootstrap.com/components/#input-groups-basic) to make it clear it is for currency? This would drop the need to parse out the `$` in the back end, and might look better visually.